libdnf-cli/output/advisorysummary.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "libdnf-cli/output/advisorysummary.hpp"
+
+#include "libdnf-cli/output/key_value_table.hpp"
+
+#include <iostream>
+
+namespace libdnf::cli::output {
+
+
+void print_advisorysummary_table(const libdnf::advisory::AdvisoryQuery & advisories, const std::string & mode) {
+    KeyValueTable output_table;
+    std::cout << mode << " advisory information summary:" << std::endl;
+
+    auto bugfixes = advisories;
+    bugfixes.filter_type("bugfix");
+
+    auto enhancements = advisories;
+    enhancements.filter_type("enhancement");
+
+    auto securities = advisories;
+    securities.filter_type("security");
+
+    auto security_critical = libdnf::advisory::AdvisoryQuery(securities);
```

Comment:
Why do you make the copy with the explicit constructor call here? You're not doing that above.

Reply:
I see, interesting, I have changed it.
Thanks!